assets/airports/lrop.json

```diff
@@ -0,0 +1,670 @@
+
+
+        "GOMDI": ["N44d51m54.285", "E025d56m39.717"],
+        "TABAV": ["N44d53m22.808", "E026d58m01.012"],
+        "USIRO": ["N44d14m30.397", "E025d45m52.599"],
+        "EKSUN": ["N44d55m13.731", "E026d17m08.121"],
+        "ELKAV": ["N44d55m43.219", "E026d51m38.107"],
+        "ETUXA": ["N44d49m34.939", "E026d17m57.475"],
+        "OP881": ["N44d35m05.531", "E026d11m36.734"],
+        "OP882": ["N44d34m24.940", "E026d11m42.684"],
+        "OP883": ["N44d35m42.748", "E026d19m57.990"],
+        "OP884": ["N44d35m02.161", "E026d20m03.842"],
+        "OP885": ["N44d37m04.432", "E026d38m48.993"],
+        "OP886": ["N44d36m19.747", "E026d37m56.725"],
+        "OP887": ["N44d37m54.519", "E026d50m45.222"],
+        "OP888": ["N44d37m06.677", "E026d49m05.656"],
+        "OP889": ["N44d21m59.577", "E026d21m56.214"],
+        "OP890": ["N44d46m02.123", "E026d10m00.159"],
+        "OP891": ["N44d49m16.707", "E026d02m38.662"],
+        
+        "ENIMA": ["N45d04m46.00","E24d54m09.00"],
+        "PELUR": ["N45d17m12.00","E25d29m18.00"],
+        "LAPKA": ["N45d17m34.00","E26d30m33.00"],
+        "URELA": ["N45d29m48.00","E26d33m40.00"],
+        "RIVOS": ["N44d39m21.00","E27d39m17.00"],
+        "DIRAL": ["N44d30m39.00","E27d33m15.00"],
+        "KOMAN": ["N43d59m00.00","E26d13m00.00"],
+        "ARGES": ["N44d04m56.00","E26d49m36.00"],
+        "RASUB": ["N43d58m44.00","E25d25m06.00"],
+        "ORTIP": ["N43d58m40.00","E25d19m59.00"],
+        "ELVAB": ["N44d09m52.00","E25d05m27.00"],
+        "VIKBI": ["N44d15m31.00","E24d39m21.00"],
+        "UBOGU": ["N44d45m47.00","E24d40m06.00"],
+        "VAMON": ["N44d23m58.00","E24d40m47.00"],
+        "BULEN": ["N43d45m00.00","E25d49m00.00"],
+        "PEMOK": ["N44d21m52.90","E24d20m54.50"],
+        "LAMIT": ["N45d06m14.00","E23d22m29.00"],
+        "KODRU": ["N46d11m32.00","E28d07m26.00"],
+        "FOCSA": ["N45d59m41.00","E26d41m23.00"],
+        "SOMOV": ["N43d42m00.00","E24d51m00.00"],
+
+        "BIVBU": ["N44d55m14.00","E24d19m54.00"] 
+
+    },
+    "runways": [
+        {
+            "name":        ["08R", "26L"],
+            "end": [
+                           ["N44.564661", "E26.076747", "314ft"],
+                           ["N44.56797", "E26.120428", "314ft"]
+                         ],
+            "delay":       [2, 2],
+            "sepFromAdjacent": [2.5, 2.5],
+            "ils":         [true, true]
+        },
+        {
+            "name":        ["08L", "26R"],
+            "end":         [                
+                           ["N44.576622", "E26.0841", "313ft"],
+                           ["N44.579925", "E26.127764", "313ft"]
+                         ],
+            "delay":       [2, 2],          
+            "sepFromAdjacent": [2.5, 2.5],
+            "ils":         [true, true]
+        }
+    ],
+    "sids": {   
+        "SOKRU1K": { 
+	        "icao": "SOKRU1K",       
+	        "name": "Sokru One Kilo",
+	        "suffix": {"08R":"", "08L":""},
```

Comment:
`TOSVI` arrival for 26 L/R has 2 separate procedures in case of high traffic `TOSVI3X` and `TOSVI1R`, how will I specify 2 procedures for same runway coming from the same entry point?